# HG changeset patch
# User Andrew Cooper <andrew.cooper3@citrix.com>
# Date 1403104968 -3600
# Node ID c97e3bc6f034d3fc1817dafe13b925eefaa19b61
# Parent  12665c8944450c1764a925cfdbd3c2224b93a1cd
tools/pygrub: Fix extlinux when /boot is a separate partition from /

Grub and Grub2 already cope with this.

Reported-by: Joseph Hom <jhom@softlayer.com>
Signed-off-by: Andrew Cooper <andrew.cooper3@citrix.com>
CC: Ian Campbell <Ian.Campbell@citrix.com>
CC: Ian Jackson <Ian.Jackson@eu.citrix.com>
Acked-by: Ian Campbell <ian.campbell@citrix.com>

diff -r 12665c894445 -r c97e3bc6f034 tools/pygrub/src/pygrub
--- a/tools/pygrub/src/pygrub
+++ b/tools/pygrub/src/pygrub
@@ -418,7 +418,9 @@ class Grub:
                    map(lambda x: (x,grub.ExtLinuxConf.ExtLinuxConfigFile),
                        ["/boot/isolinux/isolinux.cfg",
                         "/boot/extlinux/extlinux.conf",
-                        "/boot/extlinux.conf"]) + \
+                        "/boot/extlinux.conf",
+                        "/extlinux/extlinux.conf",
+                        "/extlinux.conf"]) + \
                    map(lambda x: (x,grub.GrubConf.GrubConfigFile),
                        ["/boot/grub/menu.lst", "/boot/grub/grub.conf",
                         "/grub/menu.lst", "/grub/grub.conf"])
